content/en/docs/architecture/retester.md

```diff
@@ -40,5 +55,20 @@ We can enable the retester on a repository by adding a flag `--enable-on-repo=<o
         -repo:openshift/ci-tools                  # Exclude repo openshift/ci-tools
 {{< / highlight >}}
 
+### Config file
+There are 3 levels of policy in the configuration file: retester (global), org, repo. The policy has 3 attributes: `enabled`, `max_retests_for_sha`, `max_retests_for_sha_and_base`.
```

Comment:
```suggestion
There are 3 levels of policy in the configuration file: retester (global), org, repo. If a field of policy is defined at two levels, the value at the more specific level precedes. The values in the more generic level are inherited if they are not defined in the more specific level.

```yaml
`enabled`:
  - `true`: enable retester
  - `false`: disable retester
  - not defined: use the value in the above level
`max_retests_for_sha_and_base`: the number of retesting for each sha and base
`max_retests_for_sha`: the number of retesting for each sha
```

```

Reply:
Should be OK without the paragraphs below.